src/IIIF/IIIF/Presentation/V3/ResourceBase.cs

```diff
@@ -112,6 +112,13 @@ public abstract class ResourceBase : JsonLdBase, IResource
     /// </summary>
     [JsonProperty(Order = 31)]
     public List<string>? Behavior { get; set; }
+    
+    /// <summary>
+    /// A resource used to hold data on a geographic location.  This follows the GeoJSON spec.
+    /// See <a href="https://iiif.io/api/extension/navplace/">navplace</a>
+    /// </summary>
+    [JsonProperty(Order = 500)]
+    public NavPlace.NavPlace? NavPlace { get; set; }
```

Comment:
`ResourceBase` seems too wide a class to put this on, only Collection/Manifest/Range/Canvas should have it. 

> Other types of resource MUST NOT have the navPlace property.

but `ResourceBase` adds to ImageServices, SearchServices, Annotations etc. 

`StructureBase` is more appropriate as that covers the Collection/Manifest/Range/Canvas types